Resources/doc/configuration.rst

```diff
@@ -758,6 +758,7 @@ the ORM resolves to:
             metadata_cache_driver: ~
             query_cache_driver: ~
             result_cache_driver: ~
+            naming_strategy: doctrine.orm.naming_strategy.default
 
 There are lots of other configuration options that you can use to overwrite
```

Comment:
First, the sentence above that code paragraph says "This following configuration example shows **all** the configuration defaults". As far as I understand this means that all options with any defaults should be listed.

Secondly, I cannot find any explanation about this Option in this file (https://www.doctrine-project.org/projects/doctrine-bundle/en/2.2/configuration.html). Maybe I am overseeing anything. Please correct me if I am wrong.

Reply:
> Secondly, I cannot find any explanation about this Option in this file ([doctrine-project.org/projects/doctrine-bundle/en/2.2/configuration.html](https://www.doctrine-project.org/projects/doctrine-bundle/en/2.2/configuration.html)). Maybe I am overseeing anything. Please correct me if I am wrong.

Alright. explained -> mentioned

> First, the sentence above that code paragraph says "This following configuration example shows all the configuration defaults". As far as I understand this means that all options with any defaults should be listed.

Then that text is wrong. There are lot of other configuration options not specified there, but they are in huge code block before. Surely purpose of second code block wasn't to specify all of them again.

Reply:
I think it's not mentioned because defaults are fine for most people. And also because this is a doctrine bundle, not doctrine. Bundle is just a bridge for ORM. Options should be primarily explained in ORM docs, we shouldn't have to repeat explanations here. So I would suggest to improve docs of ORM primarily.

Reply:
Okay, this makes sense. Thanks for your patience ;-)